src/bci_build/package/kiwi.py

```diff
@@ -36,16 +37,19 @@
             "iproute2",
             "jing",
             "kiwi-systemdeps-core",
+            "kpartx",
             "libxml2-devel",
             "lvm2",
             "make",
             "netcat-openbsd",
             "python3-devel",
             "python3-kiwi",
             "python3-pip",
+            "squashfs",
```

Comment:
```suggestion
            "kiwi-systemds-disk-images",
            "libxml2-devel",
            "lvm2",
            "make",
            "netcat-openbsd",
            "python3-devel",
            "python3-kiwi",
            "python3-pip",
```

Reply:
that doesn't exist here. we can use kiwi-systemdeps-filesystems though hopefully